src/marketing/marketing-menu.md

```diff
@@ -27,7 +27,7 @@ Create [related product rules]({{ site.baseurl }}{%- link marketing/product-rela
 
 ### Private Sales
 
-[Private Sales]({{ site.baseurl }}{%- link marketing/promotions.md -%}) and other catalog events are a great way to leverage your existing customer base to generate buzz and new leads with exclusive access for members only, or by invitation.
+Private Sales and other catalog events are a great way to leverage your existing customer base to generate buzz and new leads with exclusive access for members only, or by invitation.
```

Comment:
Why did you remove the link here? I just meant make the text "Private Sales" but keep the link. Apologies for the misunderstanding.

Reply:
> Why did you remove the link here?

Good question.... Probably I am slowdown today...

**fixed**

Please review!

Thank you, @shrielenee